atuin-server/src/handlers/user.rs

```diff
@@ -43,6 +43,70 @@ pub async fn get(
     })
 }
 
+pub async fn delete(
+    login: LoginRequest<'_>,
+    db: impl Database + Clone + Send + Sync,
+) -> JSONResult<ErrorResponseStatus<'static>> {
+    let user = match db.get_user(login.username.borrow()).await {
```

Comment:
could we change this to use the currently-logged-in user session please? rather than taking a login request